client/python/openlineage/client/client.py

```diff
@@ -228,6 +228,13 @@ def _http_transport_from_env_variables() -> HttpTransport:
         if endpoint is not None:
             config.endpoint = endpoint
 
+        headers_str = os.environ.get("OPENLINEAGE_HTTP_HEADERS")
+        if headers_str is not None:
+            for val in headers_str.split(","):
```

Comment:
I would use `;`, as the commit description suggests

Reply:
Thanks for catching this! I've updated to `;`